op-batcher/batcher/channel_config_test.go

```diff
@@ -0,0 +1,39 @@
+package batcher_test
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-batcher/batcher"
+	"github.com/stretchr/testify/require"
+)
+
+// TestInputThreshold tests the [ChannelConfig.InputThreshold] function.
+func TestInputThreshold(t *testing.T) {
```

Comment:
This test would be a good candidate for table tests.
We only care about `TargetFrameSize, TargetNumFrames, & ApproxComprRatio` here & we have a known output (for each input set) so it's easy to test different combos like `TargetNumFrames > 1`